src/precice/SolverInterface.cpp

```diff
@@ -74,7 +74,12 @@ bool SolverInterface::isWriteDataRequired(
 
 bool SolverInterface::isTimestepComplete() const
 {
-  return _impl->isTimestepComplete();
+  return _impl->isTimeWindowComplete();
```

Comment:
Like mentioned in https://github.com/precice/precice/issues/602#issuecomment-571979989, we should also consider throwing a "deprecated" warning here.

Reply:
Or remove it completely, if we also merge #614, for consistency and since the API will break eitherway. @fsimonis

Reply:
I'm fine with both approaches. Just referring to what I thought we agreed on in #602.

Reply:
Let's try to find a conclusion here: https://github.com/precice/precice/issues/602#issuecomment-576380396